cni/pkg/plugin/plugin.go

```diff
@@ -239,7 +239,7 @@ func CmdAdd(args *skel.CmdArgs) (err error) {
 				log.Errorf("istio-cni cmdAdd failed to check ambient: %s", err)
 			}
 		}
-		if !added && !excludePod {
+		if !added {
```

Comment:
I see, the "!excludePod" here is not redundant, it's neccesary, so close this PR.